ros_buildfarm/templates/doc/doc_independent_docker_job.xml.em

```diff
@@ -123,6 +133,22 @@ doc_repository_name = os.path.splitext(os.path.basename(doc_repository_url))[0]
         'echo "# END SECTION"',
     ]),
 ))@
+@[if upload_host is not None]@
+@(SNIPPET(
+    'builder_shell',
+    script='\n'.join([
+        'if [ -d "$WORKSPACE/repositories/%s/build/html" ]; then' % (doc_repository_name),
+        '  echo "# BEGIN SECTION: rsync API documentation to server"',
+        '  ssh %s@%s "mkdir -p %s"' %
+          (upload_user, upload_host, os.path.join(upload_root, 'build', 'html')),
+        '  cd $WORKSPACE/repositories/%s/build' % (doc_repository_name),
+        '  rsync -e ssh --stats -r --delete html %s@%s:%s' % \
+          (upload_user, upload_host, os.path.join(upload_root, 'build/html')),
+        '  echo "# END SECTION"',
+        'fi',
+    ]),
+))@
+@[end if]@
```

Comment:
This is all new here because this job previously only implemented support for `upload_repository_url` right?

Reply:
Exactly.